src/main/kotlin/com/kotlin/boardproject/MyDataInit.kt

```diff
@@ -1,14 +1,14 @@
 package com.kotlin.boardproject
 
-import com.kotlin.boardproject.auth.ProviderType
-import com.kotlin.boardproject.common.enums.NormalType
-import com.kotlin.boardproject.common.enums.Role
-import com.kotlin.boardproject.model.Comment
-import com.kotlin.boardproject.model.NormalPost
-import com.kotlin.boardproject.model.User
-import com.kotlin.boardproject.repository.CommentRepository
-import com.kotlin.boardproject.repository.NormalPostRepository
-import com.kotlin.boardproject.repository.UserRepository
+import com.kotlin.boardproject.global.enums.ProviderType
```

Comment:
🚫 **[ktlint]** <standard:import-ordering> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Imports must be ordered in lexicographic order without any empty lines in-between with "java", "javax", "kotlin" and aliases in the end